src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  const sum = (x) => {
+    if (typeof x !== 'undefined') {
```

Comment:
you don't need 'typeof' here
```suggestion
    if (x !== undefined) {
```